docs/compatibility.md

```diff
@@ -256,7 +256,6 @@ The formats which are supported on GPU and 100% compatible with Spark are :
 
 Examples of supported formats that may produce different results are:
 
-- Use of a two digit year, i.e.: `yy`, may use a different century on GPU compared to CPU
```

Comment:
Should this be updated to explain that yy is not supported when pasting a string, but in other cases it is?

Reply:
This line was removed from the documentation for parsing strings, and I have now added a new section for" formatting dates as strings" and have pointed out that "yy" is supported for formatting but not for parsing.